fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidHavingBuilder.java

```diff
@@ -15,16 +15,22 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.List;
-import java.util.Map;
+import java.util.ArrayList;
 import java.util.Set;
+import java.util.Map;
+import java.util.Arrays;
+
 import java.util.stream.Collectors;
 
+import static com.yahoo.bard.webservice.web.ErrorMessageFormat.HAVING_OPERATOR_IMPROPER_RANGE;
+import static com.yahoo.bard.webservice.web.ErrorMessageFormat.HAVING_OPERATOR_WRONG_NUMBER_OF_PARAMETERS;
+
 /**
  * Class to hold static methods to build druid query model objects from ApiHaving.
  */
 public class DruidHavingBuilder {
     private static final Logger LOG = LoggerFactory.getLogger(DruidHavingBuilder.class);
-
+    private static final int HAVING_PARAM_LENGTH = 2;
```

Comment:
I actually gave you bad advice on the variable name.  There are at least three having signatures that make sense, >1 (ins), exactly 2 (ranges), and 1 (comparisons).
Please change this to reflect that it's range parameters.  As in HAVING_RANGE_PARAM_LENGTH.

I'm +1ing the PR because it's a simple change. 

You just need one more review from someone.